test/build.gradle

```diff
@@ -1,188 +1,26 @@
-buildscript {
-  repositories {
-    maven { url "https://plugins.gradle.org/m2/" }
-    jcenter()
-  }
-
-  dependencies {
-    classpath 'org.codehaus.groovy:groovy-json:2.4.15'
-    classpath 'org.codehaus.groovy.modules.http-builder:http-builder:0.5.2'
-    classpath "com.moowork.gradle:gradle-node-plugin:1.2.0"
-    classpath "net.saliman:gradle-properties-plugin:1.4.6"
-    classpath "com.marklogic:ml-gradle:3.3.0"
-  }
-}
-
-apply plugin: 'java'
-apply plugin: 'net.saliman.properties'
-apply plugin: "com.moowork.node"
-apply plugin: "com.marklogic.ml-gradle"
-
-ext {
-  mlDeployerAppName = "MarkLogic-Esri-Connector"
-  mlDeployerDir = "build/${mlDeployerAppName}"
-  mlDeployerGroup = "ML Deployer"
-  mlDeployerZipFilename = "${mlDeployerAppName}.zip"
+plugins {
```

Comment:
This file is now just focused on compiling and running the JUnit tests. Though, in DEVEXP-222, I plan on adding exec tasks for running each variation of the Koop server with a different auth.